doc/underpinnings.tex

```diff
@@ -0,0 +1,779 @@
+\documentclass[margin=5mm]{article}
+\usepackage{tikz}
+\usepackage{parskip}
+\usepackage{listings}
+\usepackage{color}
+\usepackage{amsmath}
+
+
+\usepackage[utf8]{inputenc}
+\usepackage[english]{babel}
+ \usepackage{hyperref}
+\hypersetup{
+    colorlinks=true,
+    linkcolor=blue,
+    filecolor=magenta,      
+    urlcolor=cyan,
+}
+ 
+\urlstyle{same}
+
+\usetikzlibrary{matrix,calc}
+\definecolor{dkgreen}{rgb}{0,0.6,0}
+\definecolor{gray}{rgb}{0.5,0.5,0.5}
+\definecolor{mauve}{rgb}{0.58,0,0.82}
+
+\lstset{frame=tb,
+  aboveskip=3mm,
+  belowskip=3mm,
+  showstringspaces=false,
+  columns=flexible,
+  basicstyle={\small\ttfamily},
+  numbers=none,
+  numberstyle=\tiny\color{gray},
+  keywordstyle=\color{blue},
+  commentstyle=\color{dkgreen},
+  stringstyle=\color{mauve},
+  breaklines=true,
+  breakatwhitespace=true,
+  tabsize=3
+}
+
+
+\begin{document}
+
+\section{Send Population Model}
+
+From Simulation, Modeling and Analysis:
```

Comment:
put book title in italics and add footnote or author name + page number in brackets so people can look it up if they want